program/Cargo.toml

```diff
@@ -52,14 +52,17 @@ cfg-if = "1"
 demo-meta = { path = "../examples/binaries/meta" }
 
 [build-dependencies]
-gear-runtime = { path = "../runtime/gear" }
+gear-runtime = { path = "../runtime/gear", optional = true }
+vara-runtime = { path = "../runtime/vara", optional = true }
 frame-metadata = "15.0.0"
 parity-scale-codec = "3.1.5"
 subxt-codegen = "0.22.0"
 syn = "1"
 
 [features]
-default = [ ]
+default = [ "gear" ]
+gear = [ "gear-runtime" ]
+vara = [ "vara-runtime" ]
```

Comment:
Here and for `client` should be duplicated to runtime features-set:
`lazy-pages`, `debug-data`, etc, that throws into runtime

Also disable default features for runtimes (std stuff)

Reply:
This may be the reason why metadata can't be build directly from runtime: recheck, please

Reply:
> > This may be the reason why metadata can't be build directly from runtime: recheck, please
> 
> what you mean about this? we can build the metadata from runtime directly since the `check` CI works https://github.com/gear-tech/gear/actions/runs/3262279860/jobs/5359139117

Didn't get the point here

Reply:
> > Also disable default features for runtimes (std stuff)
> 
> can not build with `default-features = false` https://github.com/gear-tech/gear/actions/runs/3265494330/jobs/5367802193, I think it's free to choose `std` since the cli is based on std itself

Yeah, ok. We found that `substrate_wasm_builder` cuts `std` from runtime **always**

Reply:
> yeah, `std`, `runtime-benchmarks` are unnecessary since the runtime apis will not change with these features, but I'm not familiar with the `try-runtime` feature

If I'm not mistaken `runtime-benchmarks` adds some extrinsics, so api should change. Same for `try-runtime`? I see the reason just to paste it there for future purposes. Nobody got hurt from these extra unused features